### PR TITLE
CLI: Don't display inactive stake lockups

### LINKED ISSUE #10265: Clean up Lockup display when viewing stake account on CLI
#### Problem
As lockups will at some point be a thing of the past, and most users will never encounter or use a locked stake account, the three lockup lines at the end of a stake account display my cause unneeded confusion.

#### Proposed Solution
For stake accounts that are not locked, meaning the lockup date or epoch is in the past, do not display the lines Lockup Date, Lockup Epoch, Lockup Custodian.

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

CLI displays lockup parameters for stake accounts with no lockout in effect

#### Summary of Changes

Only display lockups for which `Lockup::is_in_force()` is `true`

Fixes #10265

### CODE REVIEW

File: cli/src/stake.rs [hunk 1]
```diff
@@ -1279,11 +1282,15 @@ pub fn build_stake_state(
             },
             stake,
         ) => {
-            // The first entry in stake history is the previous epoch, so +1 for current
-            let current_epoch = stake_history.iter().next().unwrap().0 + 1;
+            let current_epoch = clock.epoch;
```
Comment (CriesofCarrots) [inline cli/src/stake.rs:1285]: 🎉

File: cli/src/cli_output.rs [hunk 2]
```diff
@@ -516,19 +516,21 @@ impl fmt::Display for CliStakeState {
...
+                    f,
+                    "Lockup Timestamp: {} (UnixTimestamp: {})",
+                    DateTime::<Utc>::from_utc(
+                        NaiveDateTime::from_timestamp(lockup.unix_timestamp, 0),
+                        Utc
+                    )
+                    .to_rfc3339_opts(SecondsFormat::Secs, true),
+                    lockup.unix_timestamp
+                )?;
+                writeln!(f, "Lockup Epoch: {}", lockup.epoch)?;
```
Comment (CriesofCarrots) [inline cli/src/cli_output.rs:531]: We could be a little more-smarter here and only display the timestamp or epoch field if it's not default, which might be less confusing to users when only one is set?

Comment (t-nelson) [reply cli/src/cli_output.rs:531]: I can't argue with more-smarter! 😉

Comment (t-nelson) [reply cli/src/cli_output.rs]: 📈 🧠 https://github.com/solana-labs/solana/pull/10278/commits/4e03938a4401e3002907cdc21efd9c4d3224071b?w=1#diff-506d0550bb762fa51761fd1784b4c53fR521-R535

**Pull request merged**